jp.js

```diff
@@ -888,7 +888,7 @@ social.trades.message トレードメッセージを設定から有効にして
 social.trades.warning このトレードではあなたは何も手に入れません!
 social.trades.accept 本当にこの申請を許可しますか?
 stats.airdrops KRパッケージ
-stats.airdrops.stolen Airdrops Stolen
+stats.airdrops.stolen KRパッケージが盗まれました
```

Comment:
hmm.

別の言葉を使ってくれますか？なんか「KRパッケージ」と「Airdrops Stolen」の意味が基本的に違うと思います。

Reply:
`stats.airdrops`を"KRパッケージ"として翻訳していたので、今の所それに合わせています。
もしほかの言葉に変えるのであれば、`stats.airdrops`も修正するべきだと思っています。

Reply:
that's fair

Reply:
I talked about this sentence with NAVY8, we think almost Japanese players don't call it "エアドロップ" but "KRパッケージ" so I wanna leave it as it is.

Reply:
@Hiro527 the issue is that the game references the statistic as "Airdrops", not "KR package". While it might make sense changing it to what Japanese players call the statistic, it'll make it non-uniform with English and other languages.

Reply:
Okay I checked it's shown as "Airdrops"
To make it uniform with other languages, I'll change it into "エアドロップ"
Thank you for letting me know🙏

Reply:
Fixed: de87302